content/en/docs/setup/upgrade/canary/index.md

```diff
@@ -261,6 +261,13 @@ with the canary uninstall.
 
 ## Cleanup
 
+1. Clean up created revisioned tags:
+
+   {{< text bash >}}
+   $ istioctl tag remove prod-stable
+   $ istioctl tag remove prod-canary
+   {{< /text >}}
```

Comment:
```suggestion
    {{< text bash >}}
    $ istioctl tag remove prod-stable
    $ istioctl tag remove prod-canary
    {{< /text >}}
```

Reply:
Thanks, the linter told me the same, and I think this has been fixed.